src/stream.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const logger = require('./logger');
+const groupBy = require('lodash.groupby');
+const AWS = require('aws');
+const allowedEvents = ['MODIFY', 'INSERT'];
 
 module.exports.handler = async (event) => {
-	logger.debug({event}, 'Event received');
+	const unmarshalledRecords = event.Records
+		.filter(record => record.eventName && allowedEvents.includes(record.eventName))
```

Comment:
Minor: You don't need the explicit check for `record.eventName`. `allowedEvents.includes()` will happily return false if given an undefined value. I just double checked this to make sure.

Reply:
Nice!